.env

```diff
@@ -0,0 +1,4 @@
+DB_HOST=localhost
```

Comment:
The best practice is not to commit this file (need to add to gitignore) as there might be sensitive data down the road. Instead, commit `env.sample` and add instructions to rename this file `.env` in the project instructions. 

I saw further down in the config that you plan to have `.env.dev.local` or `.env.test.local`. But I would say that `.env`, `.env.sample` and `.env.test` would be more standard 💭

Reply:
Fixed in 117c274a104619052218a8225fce664c42eb8ad4 49fe981cc29244dffd1151c36206e9a0c7b94f3b